dlang.org.ddoc

```diff
@@ -27,10 +27,10 @@ _=
 
 COMMON_SCRIPTS =
     $(SCRIPTLOAD https://ajax.googleapis.com/ajax/libs/jquery/1.7.2/jquery.min.js)
-    $(SCRIPT window.jQuery || document.write('<script src="$(ROOT)/js/jquery-1.7.2.min.js">\x3C/script>'))
-    $(SCRIPTLOAD $(ROOT)/js/codemirror-compressed.js)
-    $(SCRIPTLOAD $(ROOT)/js/run.js)
-    $(SCRIPTLOAD $(ROOT)/js/cssmenu.js)
+    $(SCRIPT window.jQuery || document.write('<script src="$(ROOT_DIR)js/jquery-1.7.2.min.js">\x3C/script>'))
```

Comment:
This business with `$(ROOT_DIR)` including the slash is distinctly not awesome. A slash is always easy to add but more difficult to remove. Can we please go the other way, thanks.